apoc/_table_row_classifier.py

```diff
@@ -155,16 +168,39 @@ def _prepare_feature_table(
             The table from which to make the prediction. Each row of the table
             will be classified. The table can either be a pandas DataFrame or a
             Dict with string keys (column names) and numpy array columns.
+        ground_truth : np.array
+            The array containing the ground truth class for each row in feature_table
 
         Returns
         -------
         ordered_features : List[np.ndarray]
             The features stored in a list. The order of the features is
             specified by self.ordered_feature_names
         """
+        # make sure it's a DataFrame and we can modify it
+        feature_table = pd.DataFrame(feature_table).copy()
+
+        # store original keys
+        original_keys = feature_table.keys().copy()
+
+        # add ground_truth to table so that we can filter it with the other columns
+        if "ground_truth" in feature_table.keys():
+            raise ValueError("feature_table must not contain column named 'ground_truth'")
+        feature_table.insert(len(feature_table.keys()), 'ground_truth', ground_truth)
```

Comment:
This is fine, but if I understood the intent correctly, you can also do `feature_table["ground_truth"] = ground_truth` (might be more clear)